sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/PermissionImpl.java

```diff
@@ -26,11 +26,11 @@ public List<String> notActions() {
 
     @Override
     public List<String> dataActions() {
-        return innerModel().dataActions();
+        return innerModel().actions();
```

Comment:
@weidongxu-microsoft dataActions and notDataActions are incorrectly set to actions and notActions.